src/cgl.rs

```diff
@@ -103,6 +137,7 @@ pub const CORE_INTEGER_ATTRIBUTES: &'static [CGLPixelFormatAttribute] =
       kCGLPFAOpenGLProfile,
       kCGLPFAVirtualScreenCount];
 
+#[link(name = "OpenGL", kind = "framework")]
```

Comment:
I got missing symbols trying it with glutin for some reason, linking to the OpenGL framework where the CGL stuff is solved it. Might be because servo (which i'm assuming this is used for) already links it via `-framework OpenGL` in some other crate?